hr_payroll_account_operating_unit/models/account_move.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L. -
+# Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, models
+
+
+class AccountMove(models.Model):
+
+    _inherit = 'account.move'
+
+    @api.model
+    def create(self, vals):
```

Comment:
@aheficent IMO this method should be in account_operating_unit module since there is a possibility that the method will be use by other module.